docs/projections.md

```diff
@@ -0,0 +1,170 @@
+# Projections
+
+New in v7 are queries and projectons.
+
```

Comment:
Reading this document I've come to realise that a query is the actually the same thing as a projection but without persistence of state.... Usually one might think of a query as a representation of (pre) calculated state. I thought for example that queries are a way to retrieve persisted projection state.

I think it would be good to add a small section here explaining this subtlety. Looping over a (sub)set of events (a stream) and updating a running state while you do so. Projections stores state, query executed each time.